pepper-apis/dss-core/src/main/java/org/broadinstitute/ddp/db/dao/JdbiDateQuestionFieldOrder.java

```diff
@@ -6,10 +6,14 @@
 import org.jdbi.v3.sqlobject.statement.SqlUpdate;
 import org.jdbi.v3.stringtemplate4.UseStringTemplateSqlLocator;
 
-@UseStringTemplateSqlLocator
+
 public interface JdbiDateQuestionFieldOrder extends SqlObject {
 
+    @UseStringTemplateSqlLocator
     @SqlUpdate("insert")
     @GetGeneratedKeys
     long insert(@Bind("questionId") long questionId, @Bind("dateFieldTypeId") long dateFieldTypeId, @Bind("displayOrder") int displayOrder);
+
+    @SqlUpdate("DELETE FROM date_question_field_order WHERE date_question_id=:questionId")
```

Comment:
Adds whitespace to improve readability.

```suggestion
    @SqlUpdate("DELETE FROM date_question_field_order WHERE date_question_id = :questionId")
```